src/main/java/pipe/gui/petrinet/graphicElements/tapn/TimedPlaceComponent.java

```diff
@@ -36,7 +36,7 @@ public class TimedPlaceComponent extends Place {
     private dk.aau.cs.model.tapn.TimedPlace place;
     private final dk.aau.cs.model.tapn.event.TimedPlaceListener listener = timedPlaceListener();
```

Comment:
These `isHeadless` checks may be interesting to add to main, as they allow the frontend to be run in a headless environment.